apps/web/src/Pages/Staking/Lockup/Lockup.tsx

```
@@ -139,28 +274,26 @@ function Lockup({
-        Number(amount) >
-          microalgosToAlgos(
-            new CoreAccount(accountInfo).availableBalance() - 2e6
-          ))
+        Number(amount) > microalgosToAlgos(availableBalance - txnCost))
     );
   }, [amount]);
 
   const errorMessage = useMemo(() => {
+    if (availableBalance <= txnCost) return "No balance available";
     if (error) {
       if (isNaN(Number(amount))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)